pages/features/list.md

```diff
@@ -1,9 +1,20 @@
 # Features
 
+Features as of Beta v0.5.3 Prerelease 1
+
 ## General/Miscellaneous Features
 
 General and Miscellaneous features.
 
+### Mod Checker
+
+When enabled, Mod Checker automatically will check your mods list for outdated, suspicous, or unknown mods. These mods may not be on the list of verified mods, and if you believe a mod is being falsely flagged, please report it in the PSS discord (``/pss``)
+<img width="750" alt="18b5977a-c6b6-4f83-80bf-61b6a1297946" src="https://github.com/FlagHater/partly-sane-skies-FlagHater/assets/106500506/24d3ee14-bc5e-42e7-8604-c2811216b457">
+
+### Privacy
+
+This toggle blocks other mods wanting to send data to their servers. Currently supported mods are Essentials & Dungeon Guide. If you know more mods that send that kind of data, feel free to report that in our discord server. This is a privacy feature, and is off by default.
```

Comment:
> This is a privacy feature, and is off by default.

Feels very weird

Reply:
I just copied Su's description from the mod, I don't know how to rephrase that, unless something like ``This feature is meant as a privacy feature, but/and is off by default``

Reply:
It's not off by default

Reply:
Someone spreading misinformation in releases?

Reply:
Where does it say that
<img width="553" alt="image" src="https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/a492c175-ea76-46f8-88a8-5fafc2066e4f">

Reply:
<img width="633" alt="image" src="https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/9bb6db2d-f472-4db6-9467-484ae4aa284d">
In disc as well

Reply:
Oh wait it wasn't releases it was in features I copied it from, I think you had added it and so I just copied that

Reply:
Just remove that line, ~~also wth su you didnt do stuff with the privacy mode~~